docs/openapi.json

```diff
@@ -2190,6 +2190,13 @@
             "name": "size",
             "required": false,
             "type": "integer"
+          },
+          {
+            "description": "Filter workflows.",
+            "in": "query",
+            "name": "search",
```

Comment:
(I know we do have the "search" name already, but wouldn't it be more appropriate to call them all "filter" some such?)